assets/js/populateProducts.js

```diff
@@ -0,0 +1,59 @@
+// Import fetch
+const http = new Http();
+import { CarouselClass } from "./carouselClass.js";
+
+http.get("products").then((data) => {
+  populateCarousel(data);
+  populateProducts(data);
+});
+
+function populateCarousel(data) {
+  const carouselList = document.querySelector(".carousel-list");
+  const carouselContainer = document.querySelector("#carousel");
+  data.forEach((item, index) => {
+    const elementClass = index === 0 ? 'carousel-list__item carousel-list__item--active' : 'carousel-list__item';
```

Comment:
```suggestion
    const elementClass = `carousel-list__item${index === 0 ? ' carousel-list__item--active' : ''` ;
```

Reply:
I dont understand this solution :/

Reply:
I've suggested using a template string to avoid typing the same `carousel-list__item` twice and moved the `active` class modifier addition inside the template string with the same ternary operation. 😃  
Which part of the suggestion is not clear?  
Or do you mean that you find your version more readable? In such case, feel free to stick to your approach, as this is just a suggestion, I could have mentioned that this is just a suggestion for a different approach. 😉

Reply:
This part is not clear `carousel-list__item${index === 0} ? `

Reply:
Do you mean the expression interpolation part: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#expression_interpolation ?

Reply:
**Nesting templates** I didnt understood how this part works `${
            index === 0 ? " carousel-list__item--active" : ""
          }` but now I do. Thanks!